packages/sommelier.rb

```diff
@@ -489,11 +503,13 @@ def self.postinstall
 
       Please open a github issue at
       https://github.com/chromebrew/chromebrew/issues/new/choose
-      with the output of both
+      with the output of
        readlink -f '/sys/class/drm/renderD129/device/driver'
       and
        readlink -f '/sys/class/drm/renderD128/device/driver'
-      if sommelier does not start properly on your arm ChromeOS device.
+      and
+       tail #{CREW_PREFIX}/var/log/sommelier.log
```

Comment:
Surprised rubocop didn't complain about the indentation here.

Reply:
It doesn't care about text blocks?